### DISCUSSION TITLE
Фильтр "недоступны на Яндекс.Музыке"

### DISCUSSION THREAD

**Discussion by SillySothoth** [Chimildic/YaMuTools#67] [Идея] [answered]
В связи с тем что много музыки из Яндекса исчезло (и продолжает периодически исчезать) было бы неплохо иметь фильтр в плейлисте типа "все недоступные треки". Увидеть что именно пропало и удалить.

**Comment by kisoid** [comment, marked as answer]
https://github.com/Chimildic/YaMuTools/discussions/57

**Reply by AlexByte** [reply]
Так надо не удалять эти треки, а показать их. Добавьте создание плейлиста, содержащего недоступные треки.

**Reply by kisoid** [reply]
В итоге это и сделали вместо того, что я предлагал. Мой ответ - указание, что фактически этот реквест дублирует мой.

**Reply by AlexByte** [reply]
@kisoid как это функционал вызвать?

**Reply by kisoid** [reply]
Фильтр треков-Управление недоступными
